cron/cron.txt

```diff
@@ -0,0 +1,14 @@
+######################
+# CantusDB Cron Jobs #
+######################
+
+# Note: This is set up to run on the production server. If you want to run it on your local machine, you will need to change the paths.
+
+# min   hour    day     month   weekday command
+0       4       *       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/daily.sh
+5       4       *       *       0       bash /home/ubuntu/code/CantusDB/cron/postgres/weekly.sh
+10      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/postgres/monthly.sh
+15      4       1       1       *       bash /home/ubuntu/code/CantusDB/cron/postgres/yearly.sh
+20      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_is_last_chant_in_feast
+25      4       1       *       *       bash /home/ubuntu/code/CantusDB/cron/management/manage.sh populate_next_chant_fields
```

Comment:
Good thought to ensure these don't all run at the same time! Can we run `populate_next_chant_fields` before `populate_is_last_chant_in_feast` (we need up-to-date data on which chants follow each other before we can accurately calculate which chants end feasts)?